cmd/notation/cert/list.go

```diff
@@ -76,8 +76,8 @@ func listCerts(opts *certListOpts) error {
 	} else {
 		// List all certificates under named store namedStore, display empty if
 		// there's no such certificate
-		for _, t := range verification.TrustStorePrefixes {
-			path, err := dir.Path.UserConfigFS.GetPath(dir.TrustStoreDir, "x509", string(t), namedStore)
+		for _, t := range notationgoTruststore.Types {
+			path, err := dir.ConfigFS().SysPath(dir.TrustStoreDir, "x509", string(t), namedStore)
```

Comment:
This `err` is not handled.

Reply:
err is handled. Same applies to below comments.